### DISCUSSION TITLE
New to Github

### DISCUSSION THREAD

**Discussion by PatrGT** [a-cosmology-group/acg#34] [General]
Hi Louis,
I was a member of ACG, albeit not very active on the group. I am trying to get to grip with this Github context. I am not sure I get it all, do I need to make a request to become a member of the A cosmology group ?
If so, can you add me to the group please ?
Thanks,
Patrick

**Comment by RedshiftDrift** [comment]
No ACG membership is required on GitHub, everybody has access to every discussions.  Just post here (as you already did), follow the discussions and post comment!

Make sure you receive all notifications from the repository [https://github.com/a-cosmology-group/acg](https://github.com/a-cosmology-group/acg), the top-right menu lets you select which activity you want to follow.  To change your notification settings, click on your image (top-right) -> Settings -> Notifications and select how you want to receive notifications.

You are still on the ACG mailing list, further information about that list will be sent to you directly from Gaggle.email.
